jupyter_sparkmonitor/README.MD

```diff
@@ -0,0 +1,59 @@
+# Jupyter Notebook with Sparkmonitor
+
+This folder contains the initialization action `sparkmonitor.sh` to quickly setup and
+launch [Jupyter Notebook](http://jupyter.org/) with 
+[Sparkmonitor](https://krishnan-r.github.io/sparkmonitor/) to show Spark UI inside your notebook.
+
+Note: This init action uses Conda and Python 3. 
+
+__Pre-requisites:__
+This initialization action uses the [Jupyter Optional Component](https://cloud.google.com/dataproc/docs/concepts/components/jupyter)
+which requires Cloud Dataproc image version 1.3 and later. The Jupyter Optional
+Component's web interface can be accessed via
+[Component Gateway](https://cloud.google.com/dataproc/docs/concepts/accessing/dataproc-gateways)
+without using SSH tunnels. Also, you will need Anaconda as another Optional Component.
+
+
+## Using this initialization action
+
+You can use this initialization action to create a new Dataproc cluster with
+Sparkmonitor installed:
+
+1.  Use the `gcloud` command to create a new cluster with this initialization
+    action. The following command will create a new cluster named
+    `<CLUSTER_NAME>`.
+
+    ```bash
+    # Simple one-liner; just use all default settings for your cluster.
+    # Jupyter will run on port 8123 of your master node.
+    CLUSTER=<CLUSTER_NAME>
+    gcloud dataproc clusters create $CLUSTER \
+        --optional-components ANACONDA,JUPYTER --enable-component-gateway \
+        --initialization-actions gs://$MY_BUCKET/jupyter-sparkmonitor/sparkmonitor.sh
```

Comment:
Nit: your PR uses an underscore (`jupyter_sparkmonitor`) -- we should reconcile the two. Since the other init action directories use dashes, I'd suggest changing the file names to use dashes.